zeppelin-web/src/components/navbar/navbar.html

```diff
@@ -10,7 +10,19 @@
 WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
--->
+  -->
+<script type="text/ng-template" id="notebook_list_renderer.html">
+  <a ng-if="note.id" href="#/notebook/{{note.id}}">{{note.name || 'Note ' + note.id}} </a>
+  <li ng-if="!note.id"
+      class="dropdown-submenu">
+    <a tabindex="-1" href="#">{{note.name}}</a>
```

Comment:
Page becomes empty on click of this. Can you use `stopPropagation()` or something similar?